typings/context.d.ts

```diff
@@ -350,17 +350,6 @@ export declare class ContextMessageUpdate {
   // ------------------------------------------------------------------------------------------ //
   // ------------------------------------------------------------------------------------------ //
 
-  /**
-   * Use this method to send answers to an inline query. On success, True is returned.
-   * No more than 50 results per query are allowed.
-   * @param results Array of results for the inline query
-   * @param extra Extra optional parameters
-   */
-  answerInlineQuery(
```

Comment:
answerInlineQuery exists: https://github.com/telegraf/telegraf/blob/646476197d2c6b7e02bea0ba6a9f03da67ac2809/context.js#L176-L179

Reply:
Wait... the typings included it twice so a472f80698e6e87a5655da62327cc88af14b92dd was only deduplicating it. My bad